subgraphs/.folderslintrc

```diff
@@ -0,0 +1,28 @@
+{
+  "comments": "### comments are written like this because of lack of a better way ¯\\_(ツ)_/¯ ###",
+  "rules": [
+    "### graph build outputs and node modules ###",
+    "*/generated/**",
+    "*/build/**",
+    "*/node_modules/**",
+
+    "### generic subgraph structure ###",
+    "*/abis/**",
+    "*/protocols/*/config/deployments/*",
+    "*/protocols/*/config/templates/*",
+    "*/src/**",
+
+    "### uniswap forks structure ###",
+    "*/configurations/**",
+    "*/protocols/*/config/deployments/**",
```

Comment:
Will lint fail here without the extra asterisk?

Reply:
It doesn't! Removed :D